scripts/install2

```diff
@@ -213,6 +250,30 @@ if [[ $BOARD_REVISION == *"a02082"* || $BOARD_REVISION == *"a22082"* ]] && [ ! -
     source hostapd/install
 fi
 
+# DNS Server Settings
```

Comment:
We should create folders with `install` and `uninstall` scripts. This file only runs those install scripts.